src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -246,6 +252,15 @@ private static boolean processArgs(String[] args, Set<ArgumentFlags> flags, Set<
                 case "--p2": // NOTE: this flag is not documented in the README nor the Usage info
                     flags.add(ArgumentFlags.PROGRESSIVE_PROFILING);
                     break;
+                case "--abort":
+                    flags.add(ArgumentFlags.ABORT);
+                    if (i + 1 < args.length && !args[i + 1].startsWith("-")) {
```

Comment:
Not strictly necessary, but as this code is used in a couple places, you could use a lambda to reduce code duplication

Reply:
Done.  Make sure it's what you had in mind.